### PR TITLE
Remove custom splash and use new Android API for Splash screen

### PR DESCRIPTION
**Pull request by mapm14**
## Description

https://github.com/bumble-tech/appyx/issues/234

**Dark mode:**

**Light mode:**

## Check list

- [X] I have updated `CHANGELOG.md` if required.
- [X] I have updated documentation if required.

### CODE REVIEW

File: samples/app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -10,12 +10,21 @@
...
     </style>
 
     <style name="Theme.Appyx.NoActionBar">
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
     </style>
+
+    <style name="Theme.Appyx.Starting" parent="Theme.SplashScreen">
+        <!-- Set the splash screen background -->
+        <item name="windowSplashScreenBackground">@color/appyx_dark</item>
```
Comment (LachlanMcKee) [inline samples/app/src/main/res/values/themes.xml]: How does this work with dark/light theme like we do in the node at the moment?

File: samples/app/src/main/kotlin/com/bumble/appyx/app/MainActivity.kt [hunk 2]
```diff
@@ -22,14 +23,15 @@ import com.bumble.appyx.core.modality.BuildContext
 class MainActivity : NodeActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        installSplashScreen()
```
Comment (LachlanMcKee) [inline samples/app/src/main/kotlin/com/bumble/appyx/app/MainActivity.kt:26]: Should we use `installSplashScreen()`? I believe this is for legacy projects where you already have a splash screen. Is there not a dedicated splash screen activity in the library?

Comment (LachlanMcKee) [reply samples/app/src/main/kotlin/com/bumble/appyx/app/MainActivity.kt:26]: https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#remove_the_custom_splash_screen_activity

Comment (mapm14) [reply samples/app/src/main/kotlin/com/bumble/appyx/app/MainActivity.kt:26]: If I don't do it the app crashes (at least on my device)
It's part of the step number 4 in [here](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#migrate_your_splash_screen_implementation)

File: samples/app/src/main/res/values/themes.xml (hunk 1 above)
Comment (mapm14) [reply samples/app/src/main/res/values/themes.xml]: Good catch! I forgot to modify the theme for dark mode, is done now.

File: samples/app/src/main/kotlin/com/bumble/appyx/app/MainActivity.kt (hunk 2 above)
Comment (LachlanMcKee) [reply samples/app/src/main/kotlin/com/bumble/appyx/app/MainActivity.kt:26]: ah fair enough. I thought the API suggested that they had their own activity, and for developers to not use their own activity for this purpose.

**Review by LachlanMcKee — APPROVED** [review]

**Pull request merged**